### DISCUSSION TITLE
How to transfer filters from Chrome to Firefox

### DISCUSSION THREAD

**Discussion by ghost** [darkreader/darkreader#9951] [Q&A]
I am switching browsers from Chrome to Firefox and want to keep my blacklist of sites, but I can't find where the file is for Firefox.

Is this even possible? If so, what file should I move to where?

**Comment by Gusted** [comment]
Hi @Sowansow912!

That's possible, but you cannot move the files from Chrome to Firefox directly(the browsers use different structures to store the data). So you need to instead export the settings in Chrome and import the settings into Firefox via the Dark Reader UI:

To export:
- Click Dark Reader icon.
- Click Dev tools (in the bottom-right corner).
- Click Preview new design.
- Go back to the Dark Reader UI.
- Click Settings.
- Click Manage settings.
- Click Export

To import on Firefox, use the same steps, but instead click on Import instead of Export on the last step(this will open a new tab on Firefox and is expected behavior).

**Reply by ghost** [reply]
Hi, 

Thank you for the reply, but unfortunately, I'm having some issues.

On chrome, I can successfully export the settings, and a new .json is created in my downloads folder, but when I import the settings, after I select the .json after clicking the import button, nothing happens. (I click "Import Settings", my file explorer comes up, I select the .json and click "open", and the file explorer closes) No new tab opens, nor are the settings imported.

**Reply by Gusted** [reply]
Oh sorry, I forget I never merged the related PR(https://github.com/darkreader/darkreader/pull/4724) and instead opted for Firefox to fix this issue: https://bugzilla.mozilla.org/show_bug.cgi?id=1378527

You can workaround it:
- Open the Dark Reader popup on Firefox.
- Right click 
- Click on `view page source` (be aware this can open a white page).
- This should have opened a new tab, now remove the `view-source:` suffix from the URL in the tab.
- Now you should see the Dark Reader UI, please proceed to try and import the settings.

**Reply by ghost** [reply]
This worked, thank you!

**Reply by vjt4** [reply]
hello for this to work, does one need to stay on the "preview new design" or can we go to the old design after a succesful import? cheers!
